src/test/java/com/google/api/codegen/testdata/python_main_library.baseline

```diff
@@ -208,24 +208,25 @@ class LibraryServiceApi(object):
             scopes = self._ALL_SCOPES
         if client_config is None:
             client_config = {}
+        goog_api_client = '{}/{} {} gax/{} python/{}'.format(
+            app_name,
+            app_version,
+            self._CODE_GEN_NAME_VERSION,
+            self._GAX_VERSION,
+            platform.python_version()
+        )
+        headers = [('x-goog-api-client', goog_api_client)]
         default_client_config = json.loads(pkg_resources.resource_string(
             __name__, 'library_service_client_config.json'))
-        self._defaults = api_callable.construct_settings(
+        defaults = api_callable.construct_settings(
             'google.example.library.v1.LibraryService',
             default_client_config,
             client_config,
             config.STATUS_CODE_NAMES,
             timeout,
+            headers=headers,
             bundle_descriptors=self._BUNDLE_DESCRIPTORS,
             page_descriptors=self._PAGE_DESCRIPTORS)
-        goog_api_client = '{}/{} {} gax/{} python/{}'.format(
-            app_name,
-            app_version,
-            self._CODE_GEN_NAME_VERSION,
-            self._GAX_VERSION,
-            platform.python_version()
-        )
-        self._headers = [('x-goog-api-client', goog_api_client)]
         self.stub = config.create_stub(
```

Comment:
Local, as in, private?
We want to let people drop down to the gRPC layer if needed.